umbra-js/test/Umbra.test.ts

```diff
@@ -151,13 +151,13 @@ describe('Umbra class', () => {
       expect(umbra2.chainConfig.startBlock).to.equal(8505089);
       expect(umbra2.chainConfig.subgraphUrl).to.equal(false);
 
-      // --- Goerli ---
-      const umbra3 = new Umbra(jsonRpcProvider, 5);
```

Comment:
- in cases where we only test a single chain and it's a testnet, changing that from goerli to sepolia seems reasonable
- in cases where we test things for many/all chains, we should keep goerli for now

Reply:
Fixed in the same commit above: https://github.com/ScopeLift/umbra-protocol/pull/521/commits/37be3194d7c9d2dc67f5db05b47395ebeb1f4b8e